flyway-core/src/main/java/org/flywaydb/core/internal/dbsupport/DbSupportFactory.java

```diff
@@ -131,7 +131,7 @@ public static DbSupport createDbSupport(Connection connection, boolean printInfo
         }
 
 		//Sybase ASE support
-        if (databaseProductName.startsWith("ASE")) {
+        if (databaseProductName.startsWith("Adaptive Server Enterprise")) {
```

Comment:
My fix should now allow both names. Can you see the change in the pull
request?

On Tue, Mar 1, 2016 at 4:38 PM, Axel Fontaine <notifications@github.com>
wrote:

> In
> flyway-core/src/main/java/org/flywaydb/core/internal/dbsupport/DbSupportFactory.java
> <https://github.com/flyway/flyway/pull/1221#discussion_r54574564>:
>
> > @@ -131,7 +131,7 @@ public static DbSupport createDbSupport(Connection connection, boolean printInfo
> >          }
> >
> >  		//Sybase ASE support
> > -        if (databaseProductName.startsWith("ASE")) {
> > +        if (databaseProductName.startsWith("Adaptive Server Enterprise")) {
>
> @hakkika <https://github.com/hakkika> Can you update this PR to allow
> both the old and the new name?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/flyway/flyway/pull/1221/files#r54574564>.
>



-- 
t. Kari Häkkinen